wagtailio/search/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class SearchConfig(AppConfig):
+    name = "wagtailio.search"
+    default_auto_field = "django.db.models.AutoField"
```

Comment:
This is not needed since we don't have any models in the app

Reply:
thanks, done in 21a32065